book/working_with_strings.md

```diff
@@ -56,6 +56,10 @@ Nushell currently supports the following escape characters:
 - `\n` - newline (line feed)
 - `\t` - tab
 - `\u{X...}` - a single unicode character, where X... is 1-6 hex digits (0-9, A-F)
+ 
+To create a `\0` (`NUL`) character, you may use `\u{0}` or
+[string interpolation](#string-interpolation) with [`char`](/commands/docs/char.md),
+e.g. `(char --integer 0)`.
```

Comment:
or `char nul`, `char null_byte`, `char zero_byte` as seen in `char --list`. It may also be worth nothing that things like `\u{0}` only work when surrounded by double quotes because in nushell double quotes interpret escapes.